### PR TITLE
Query Multiple Tables

### PR DESCRIPTION
**Pull request by saadat-ali-awan**
Modify the animals table to reference foreign tables
Update queries to insert and select data

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -8,4 +8,8 @@ CREATE TABLE animals (
     neutered BOOLEAN,
     weight_kg DECIMAL
 )
-ALTER TABLE animals ADD species VARCHAR(255);
\ No newline at end of file
+ALTER TABLE animals ADD species VARCHAR(255);
+CREATE TABLE owners(id SERIAL PRIMARY KEY, full_name VARCHAR, age INT);
+CREATE TABLE species(id SERIAL PRIMARY KEY, name VARCHAR);
+ALTER TABLE animals ADD owner_id INT;
```
Comment (thecodechaser) [inline schema.sql:14]: - [ ] Please add a foreign key to the owner table.
     - You can consider using this code `ALTER TABLE animals ADD owner_id int REFERENCES owners(id);`

File: schema.sql [hunk 2]
```diff
@@ -8,4 +8,8 @@ CREATE TABLE animals (
...
+ALTER TABLE animals ALTER COLUMN species TYPE INT USING species::integer;
```
Comment (thecodechaser) [inline schema.sql]: - [ ] Please make sure that the column is called species_id and has a foreign key to the species table.
    - You can consider using this code `ALTER TABLE animals ADD species_id int REFERENCES species(id);`

File: queries.sql [hunk 3]
```diff
@@ -45,3 +45,10 @@ SELECT AVG(weight_kg) FROM animals;
 SELECT CASE WHEN (COUNT(CASE WHEN neutered THEN 1 END)) > COUNT(CASE WHEN not neutered THEN 1 END) THEN 'NEUTERED' ELSE 'NOT NEUTERED' END FROM animals;
 SELECT species, MIN(weight_kg) as "Minimum Weight", MAX(weight_kg) as "Maximum Weight" FROM animals GROUP BY species;
 SELECT species, AVG(escape_attempts) as "AVERAGE ESCAPE ATTEMPTS" from animals WHERE date_of_birth BETWEEN '1990-01-01' and '2000-01-01' GROUP BY species;
+SELECT * FROM animals WHERE owner_id=(SELECT id FROM owners WHERE full_name='Melody');
```
Comment (thecodechaser) [inline queries.sql]: - [ ] Please join the animal and owner table so the query works as expected.
  - You can consider using this code. `SELECT * FROM animals LEFT JOIN owners ON animals.owner_id = owners.id WHERE full_name = 'Melody Pond';`

File: queries.sql [hunk 4]
```diff
@@ -45,3 +45,10 @@ SELECT AVG(weight_kg) FROM animals;
...
+SELECT * FROM animals WHERE species=(SELECT id FROM species WHERE name='Pokemon');
```
Comment (thecodechaser) [inline queries.sql]: - [ ] Please join the animal and species table so the query works as expected.
  - You can consider using this code. `SELECT * FROM animals LEFT JOIN species ON animals.species_id = species.id WHERE species.name = 'Pokemon';`

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @Saadat123456👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by codepantha — APPROVED** [review]
Hi @Saadat123456 👋,

## STATUS: APPROVED ✔️  

Weldone implementing the required changes 👏 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**